scripts/leaderboard.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+import json
+import requests
+from home.models import Device
+
+
+def make_leadeboard():
```

Comment:
@benperlman when the dummydata.py script is run during bin/init, the server isn't yet running, so you can't make requests against it. It's also a much slower way to generate data.

If you follow how dummydata.py is written, it's inserting records directly into the database, and I'd recommend doing something similar...

Also note that, the dummydata.py script is used to create test data for the database that's used by the development server running on localhost. However, it is separate from the database used by the tests, which is always reset to empty. That's why, for example, the tests I wrote use the faker library to generate test data in setup before the tests run.

dummydata.py creates RANDOM data, which means the results aren't predictable. So you couldn't write tests against that data anyway.

The data you should create for tests should be predicable/deterministic so that you know what results to assert for in your tests.